src/Storefront/Resources/snippet/de_DE/storefront.de-DE.json

```
@@ -806,7 +806,13 @@
     "groupComfortFeaturesYoutubeVideo": "YouTube-Video",
     "groupMarketing": "Marketing",
     "groupMarketingDescription": "Erlaubt es Google, personenbezogene Daten für Online-Werbung und Marketing zu sammeln.",
-    "groupMarketingAdConsent": "Google Werbung und Marketing"
+    "groupMarketingAdConsent": "Google Werbung und Marketing",
+    "wishlist": {
+      "description": "Um die Wunschliste-Funktion nutzen zu können, müssen Sie das \"Wunschliste\"-Cookie akzeptieren oder sich in Ihr Konto einloggen.",
+      "title": "Cookie fehlt",
```

```suggestion
      "title": "Erforderlicher Cookie fehlt",
```

---

```suggestion
      "title": "Fehlender Cookie",
```
If you keep "missing cookie" I'd suggest this instead.